great-idea-website/css/index.css

```diff
@@ -63,4 +65,169 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+/*Navigation bar*/
+
+nav {
+	display: block;
+	text-align: center;
+}
```

Comment:
Dino it looks good man. Two things i would suggest. On these display settings like display:inline-block or display:flex . YOu can do like you did and set it directly on the children. Or you can set it on the parent once and let it get passed down to the children. that might save you some typing down the road mayb not much but css is inherited so anything on the parents the children pickup automatic. 
 2nd thing . Something i see alot is a max width and margin on these full sized web pages so that it keeps it all neat in the center if the screen gets bigger. On the body or main container you canset it to something between 800px-900px alot of times i see it at 
body{
 max-width:880px;
 margin: 0 auto;
}

Reply:
As far as line 73 I needed the parent to be block which it is by default so I didn't have to write display:block.  The children I needed to be inline-block so that they can fit on the same line. I did not want the children to inherit it so had to give each child display:inline-block; so that they fit on the same line. Hope this makes sense. As far as body change (max-width:880px; with a margin:0 auto; to the body element) I added it. It pushed some elements in the navbar but I adjusted that too.